src/main/java/com/google/cloud/tools/gradle/appengine/core/SdkDownloader.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016 Google Inc. All Right Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.google.cloud.tools.gradle.appengine.core;
+
+import java.io.File;
+
+public interface SdkDownloader {
```

Comment:
nit: might be good to call use `CloudSdk` since that is the name used in other parts of the package.